samwise.js

```diff
@@ -3,75 +3,71 @@
  *
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!